### PR TITLE
feat: LIDO-68. Signing info query

### PR DESCRIPTION
**Pull request by albertandrejev**
Add support for the signing info queries.

Depends on:
- https://github.com/neutron-org/neutron-dev-contracts/pull/38
- https://github.com/neutron-org/neutron-sdk/pull/122
- https://github.com/neutron-org/neutronjsplus/pull/19

[Test run](https://github.com/neutron-org/neutron-tests/actions/runs/7801343130)

### CODE REVIEW

File: src/testcases/run_in_band/interchain_kv_query.test.ts [hunk 1]
```diff
@@ -841,4 +890,56 @@ describe('Neutron / Interchain KV Query', () => {
...
+        queryId,
+        await getHeight(gaiaChain.sdk),
+      );
+
+      const interchainQueryResult = await getValidatorsSigningInfosResult(
+        neutronChain,
+        contractAddress,
+        queryId,
+      );
+      console.log(interchainQueryResult);
```
Comment (pr0n00gler) [inline src/testcases/run_in_band/interchain_kv_query.test.ts:942]: I don't think this test actually tests anything, it just prints something in a console

**Review by pr0n00gler — CHANGES_REQUESTED** [review]
There are no tests for unbonding delegation, please add it

**Comment by albertandrejev** [comment]
> There are no tests for unbonding delegation, please add it

@foxpy

**Comment by foxpy** [comment]
https://github.com/neutron-org/neutron-tests/actions/runs/7791362663

**Comment by foxpy** [comment]
https://github.com/neutron-org/neutron-tests/actions/runs/7800817423

**Review by pr0n00gler — APPROVED** [review]

File: src/testcases/run_in_band/interchain_kv_query.test.ts [hunk 2]
```diff
@@ -841,4 +965,163 @@ describe('Neutron / Interchain KV Query', () => {
...
+
+    test('signing info registered query data', async () => {
+      const queryResult = await getRegisteredQuery(
+        neutronChain,
+        contractAddress,
+        queryId,
+      );
+      expect(queryResult.registered_query.id).toEqual(queryId);
+      expect(queryResult.registered_query.owner).toEqual(contractAddress);
+      // XXX: I could actually check that "key" is correctly derived from contractAddress,
```
Comment (pr0n00gler) [inline src/testcases/run_in_band/interchain_kv_query.test.ts:1000]: I don't think we need this here, if it's covered in SDK unit-tests

**Review by pr0n00gler — APPROVED** [review]

**Review by quasisamurai — APPROVED** [review]

**Pull request merged**